tpc/src/serializers/xml/JaxbMedia.java

```diff
@@ -0,0 +1,217 @@
+package serializers.xml;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlTransient;
+import javax.xml.bind.annotation.XmlType;
+import java.util.List;
+
+import static data.ReprUtil.repr;
+
+@SuppressWarnings("serial")
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "JaxbMedia")
+public class JaxbMedia implements java.io.Serializable {
+	@XmlTransient
```

Comment:
I think that "@XmlType" is not compellingly needed here. But I had to add "AccessType.Field" so that Jaxb ignores the getter/setter. Otherwise Jaxb would think that we have duplicates and throws: "IllegalAnnotationExceptions". Have a look at http://stackoverflow.com/questions/10795793/jaxb-illegalannotationexceptions

I think that was the problem of the original version.